src/components/Homepage/Carousel/Item.astro

```diff
@@ -0,0 +1,58 @@
+---
+import { Image } from "astro:assets";
+import tickIcon from "../../../assets/icons/tick.svg";
+import "./ItemFrame.css";
+
+interface Props {
+  type?: "text" | "image";
+  message?: string;
+  source: string;
+  imageSrc?: ImageMetadata;
+  imageAlt?: string;
+  bottomWidth?: number;
+  padding?: number;
+}
```

Comment:
Some of these attributes are mutually exclusive, so I suggest you use something like this to avoid invalid combinations:

```suggestion
interface Item {
  type: string;
  source: string;
  bottomWidth?: number;
  padding?: number;
}

interface TextItem extends Item {
  type: "text",
  message: string;
}

interface ImageItem extends Item {
  type: "image";
  imageSrc: ImageMetadata;
  imageAlt: string;
}

type Props = TextItem | ImageItem;
```

That's assuming that `bottomWidth` and `padding` apply to both. Update accordingly if not.